apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -30,99 +30,72 @@ namespace Google.Cloud.Bigtable.V2
 {
     public abstract partial class BigtableClient
     {
-        private static string UserAgent { get; } =
+        internal static string UserAgent { get; } =
             $"cbt-csharp/v{typeof(BigtableClient).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}";
 
         // TODO: Auto-generate these if possible/easy after multi-channel support is added.
+        // TODO: Other overloads for providing a Channel or CallInvoker?
 
         /// <summary>
         /// Asynchronously creates a <see cref="BigtableClient"/>, applying defaults for all unspecified settings,
-        /// and creating a channel connecting to the given endpoint with application default credentials where
+        /// and creating channels connecting to the given endpoint with application default credentials where
         /// necessary.
         /// </summary>
-        /// <param name="clientCreationSettings">Optional. <see cref="ClientCreationSettings"/> specifying how to create
-        /// <see cref="BigtableServiceApiClient"/>s.</param>
+        /// <param name="endpoint"></param>
```

Comment:
These parameters will need documenting.

Reply:
Yes, as mentioned in the notes, I left out most doc comments for now (mostly to avoid unnecessary work if things needed to be changed). I'll add them before cleaning up the PR for final review.